salt/client/mixins.py

```diff
@@ -341,40 +343,3 @@ def print_async_event(self, suffix, event):
             event.pop('_stamp')  # remove the timestamp before printing
```

Comment:
In the case of our custom event, it got printed out without having a chance to be completed with the _stamp, causing the crash at the pop statement